ios/nativeTemplate/Info.plist

```diff
@@ -24,8 +26,40 @@
 	<string>1</string>
 	<key>LSRequiresIPhoneOS</key>
 	<true/>
+	<key>NSAppTransportSecurity</key>
+	<dict>
+		<key>NSAllowsArbitraryLoads</key>
+		<true/>
+		<key>NSExceptionDomains</key>
+		<dict>
+			<key>localhost</key>
+			<dict>
+				<key>NSExceptionAllowsInsecureHTTPLoads</key>
+				<true/>
+			</dict>
+		</dict>
+	</dict>
 	<key>NSLocationWhenInUseUsageDescription</key>
 	<string></string>
+	<key>UIAppFonts</key>
```

Comment:
Maybe if we are not including all the fonts (same as on android) then we should probably remove some here safe for `MaterialIcons` and then include `glyphicon` to the list

Reply:
This means we should commit the MaterialIcon and glyphicon.

Reply:
Good point; adding it